uvicorn/middleware/wsgi.py

```diff
@@ -188,3 +194,9 @@ def wsgi(self, environ: Environ, start_response: StartResponse) -> None:
         }
         self.send_queue.append(empty_body)
         self.loop.call_soon_threadsafe(self.send_event.set)
+
+
+try:
+    from a2wsgi import WSGIMiddleware  # type: ignore # noqa
+except ImportError:
+    pass
```

Comment:
Not the most attractive solution, but it's the best option to deprecate it without touching the config module since for testing purposes we can't reload the config module import table without breaking unrelated tests that share the config module state.

It also keeps backward compatibility with someone importing it directly in code.

Reply:
```suggestion
try:
    from a2wsgi import WSGIMiddleware  # type: ignore # noqa
except ModuleNotFound:
    WSGIMiddleware = _WSGIMiddleware
```